### ISSUE TITLE
add internal support for rmarkdown 

### ISSUE THREAD

**Issue by Reaper10** [olivierkes/manuskript#752]
you add internal support for rmarkdown 
https://github.com/rstudio/rmarkdown
https://github.com/rstudio/rmarkdown
is could make mauskript more of a wysiwyg editor.

**Comment by obw** [comment]
I don't think this is the right Tool for Manuskript, which is written in `Python`. The Tool is you suggest is written in `R`.

There are `Python` libraries which are doing the same. Also it is a not trivial Task to enhance the Editor to wysiwyg.

What I see in a first step, is something like a Preview Window / Tab. More easy to develop, also not so delicate like an complete new developed Editor.

Also something like that costs performance and as long we have the performance Issues, we should solve these first and then we should think about something like that.

Which is also something, a Author seldom need. Layout of a Book should not in the Text itself.

Technical or scientific Texts, from my perspective are better worked on with Latex directly.

**Comment by Reaper10** [comment]
R Markdown Python Engine
https://rstudio.github.io/reticulate/index.html

**Comment by dmjohnsson23** [comment]
I'm new here, but just happened to notice this issue and thought I'd give my two cents on the matter.

Personally, I prefer Manuskript's existing Markdown based editor over a WYSIWYG editor anyway; in fact that was one on the big selling points of the program to me. It makes it easy to just focus on the text and not worry about formatting. I personally feel like a person using Manuskript and trying to do formatting beyond basic bold text, italics, lists, and headers is probably using the wrong tool for the job they are trying to do anyway, and the existing editor does a fantasic job with all those things. Markdown is easy, it only takes a minute or two to learn everything most people need to know, and enables you to be really productive when writing without having to switch between the keyboard and mouse.
